src/metabase/models/action.clj

```diff
@@ -409,7 +409,6 @@
                   :native-query   :query_action.dataset_query
                   ;; workaround for actions not having revisions (yet)
                   :last-edited-at :updated_at
-                  :table-id       false
```

Comment:
This is only used for rendering, so it moved to `:render-terms`. We don't validate the keys on that map, so there's no need to mark it as omitted.